packages/trpc/server/routers/viewer/teams/listMembers.handler.ts

```diff
@@ -1,69 +1,158 @@
+import { Prisma } from "@prisma/client";
+
+import { getBookerBaseUrlSync } from "@calcom/lib/getBookerUrl/client";
 import { UserRepository } from "@calcom/lib/server/repository/user";
-import type { PrismaClient } from "@calcom/prisma";
+import prisma from "@calcom/prisma";
+import { MembershipRole } from "@calcom/prisma/enums";
 import type { TrpcSessionUser } from "@calcom/trpc/server/trpc";
 
+import { TRPCError } from "@trpc/server";
+
 import type { TListMembersInputSchema } from "./listMembers.schema";
 
-type ListMembersOptions = {
+type ListMembersHandlerOptions = {
   ctx: {
     user: NonNullable<TrpcSessionUser>;
-    prisma: PrismaClient;
   };
   input: TListMembersInputSchema;
 };
 
-export const listMembersHandler = async ({ ctx, input }: ListMembersOptions) => {
-  const { prisma } = ctx;
-  const { isOrgAdmin } = ctx.user.organization;
-  const hasPermsToView = !ctx.user.organization.isPrivate || isOrgAdmin;
+const userSelect = Prisma.validator<Prisma.UserSelect>()({
+  username: true,
+  email: true,
+  name: true,
+  avatarUrl: true,
+  id: true,
+  bio: true,
+  disableImpersonation: true,
+});
+
+export const listMembersHandler = async ({ ctx, input }: ListMembersHandlerOptions) => {
```

Comment:
Fetches all the members with pagination. earlier i was using lazyLoadMembers file